hugo/content/6_troubleshoot/611_deployment-error-codes.md

```diff
@@ -34,22 +34,23 @@ The following table lists available error codes, short descriptions, and the rec
-| DEP_E172 | compression library required for the cloud bridge is missing on the device. | re-onboard the device.</a> |
-| DEP_E173 | transport libraries required for the cloud bridge are missing on the device. | re-onboard the device.</a> |
-| DEP_E174 | cloud bridge on the device encountered multiple ROS service origins. | do not add multiple deployments with the same ROS service endpoint under the same routed network.</a> |
-| DEP_E175 | python actionlib/msgs required for the cloud bridge is missing on the device. | re-onboard the device.</a> |
-| DEP_E176 | cloud bridge encountered duplicate alias on the cloud component. | change the alias name during deployment and ensure that there is no duplication of alias name under the same routed network.</a> |
-| DEP_E177 | cloud bridge on the cloud component encountered multiple ROS service origins. | re-onboard the device.</a> |
-| DEP_E2xx | internal rapyuta.io error in the components deployed on cloud | report the issue together with the relevant details to the <a href="#" onclick="javascript:FreshWidget.show();">support team</a> |
-| DEP_E3xx | internal rapyuta.io error in the components deployed on a device | report the issue together with the relevant details to the <a href="#" onclick="javascript:FreshWidget.show();">support team</a> |
-| DEP_E4xx | internal rapyuta.io error | report the issue together with the relevant details to the <a href="#" onclick="javascript:FreshWidget.show();">support team</a> |
\ No newline at end of file
+| DEP_E163 | Application has stopped and exited unexpectedly, and crashes continuously | Debug the application using the corresponding deployment logs. |
+| DEP_E171 | Cloud bridge encountered duplicate alias on the device | Change the alias name during deployment and ensure that there is no duplication of alias name under the same routed network.</a> |
+| DEP_E172 | Compression library required for the cloud bridge is missing on the device | Re-onboard the device.</a> |
+| DEP_E173 | Transport libraries required for the cloud bridge are missing on the device | Re-onboard the device.</a> |
+| DEP_E174 | Cloud bridge on the device encountered multiple ROS service origins | Ensure that there aren't  multiple deployments with the same ROS service endpoint under the same routed network.</a> |
+| DEP_E175 | Python actionlib/msgs required for the cloud bridge is missing on the device. | Re-onboard the device.</a> |
+| DEP_E176 | Cloud bridge encountered duplicate alias on the cloud component | Change the alias name during deployment and ensure that there is no duplication of alias name under the same routed network.</a> |
+| DEP_E177 | Cloud bridge on the cloud component encountered multiple ROS service origins | Re-onboard the device.</a> |
+| DEP_317 | Failed to start | <li><ul> Ensure the current device has a subpath. </ul> <ul>Ensure that the mount path isn't empty. </ul></li> |
```

Comment:
<ul><li> Ensure the current device has a subpath. </li> <li>Ensure that the mount path isn't empty. </li></ul>
I guess it will be like this.